core-services/base-ci-images/images-ocp.yaml

```diff
@@ -173,3 +173,15 @@ spec:
       name: "builder-base:rhel-8-golang-1.13"
     referencePolicy:
       type: Local
+---
+kind: ImageStream
+apiVersion: image.openshift.io/v1
+metadata:
```

Comment:
the regular 4-stable stream has the `release.openshift.io/hasReleases` annotation. I'm assuming we need that on these too.

Reply:
The release-controller will add this tag if it's not found on release streams

Reply:
Ahh good catch, thanks brad